.github/workflows/dispatch-build-bottle.yml

```diff
@@ -40,9 +40,17 @@ jobs:
         uses: actions/github-script@v6
         with:
           script: |
+            const intelMacOSRegex = /^\d+(?:\.\d+)?$/;
             return context.payload.inputs.runner.split(",")
                                                 .map(s => s.trim())
-                                                .filter(Boolean);
+                                                .filter(Boolean)
+                                                .map(s => {
+              if (intelMacOSRegex.test(s)) // Ephemeral runners
+                return `${s}-${context.runId}-${process.env.GITHUB_RUN_ATTEMPT}`
+              else
+                return s
```

Comment:
Presumably this code is temporary until we're fully on ephemeral?

Reply:
There'll always be a branch to separate self-hosted from GitHub-hosted runners.

The way our ephemeral works is by giving each runner a unique name that's never reused. The run ID does this for us (and cannot be determined ahead of time either).

Reply:
Ok, cool. I think it'd be nice to have this code live in Homebrew/actions at some point.